### ISSUE TITLE
How can I distinguish what scrolling event type has acquired (onWheel,onClick,onScroll) FullPage.js with autoScrolling={true}?

### ISSUE THREAD

**Issue by ramimazloum871** [alvarotrigo/fullPage.js#4110]
How can I distinguish if the scrolling event has acquired because of wheel scroll event, by drag and move or by clicking the circle on FullPage.js with autoScrolling={true}?

I want to make an animation when you use the wheel to change the slide.

I want to use silentMoveTo(section, slide) if you click on the circle.

something like this

`onleave(origin, destination, direction) {
  if (event.onWheel) {
    console.log("wheel went")
    return true
  }
  if (event.onScroll) {
    console.log("onScroll ")
    return true
  }
  if (event.onClick) {
    console.log("onClick")
    return true
  }
}`

**Comment by alvarotrigo** [comment]
Thanks! Good point!
I'll mark it as a possible enhancement for future versions! 👍

**Comment by ramimazloum871** [comment]
You have mentioned in StackOverflow that I can create my own click events or use my own global variables to achieve that.

I tried to use the window.addEventListener to capture the event I want but that not possible when using autoScrolling.
Can you provide me with an example of what you mean? pier in mind I'm using React.

**Comment by alvarotrigo** [comment]
> I tried to use the window.addEventListener to capture the event I want but that not possible when using autoScrolling.

Why wouldn't that be possible. There's no problem on doing that.

**Comment by ramimazloum871** [comment]
yeah, I have been doing it wrong all the time ^_^, now it's working thanks
